addons/stock/views/stock_picking_views.xml

```diff
@@ -207,8 +207,7 @@
                             </div>
                             <field name="partner_id" nolabel="1"/>
                             <field name="picking_type_id" options="{'no_open': True}"
-                                   attrs="{'invisible': [('hide_picking_type', '=', True)], 'readonly': [('id', '!=', False)]}"
-                                   domain="context.get('restricted_picking_type_code', []) and [('code', '=', context.get('restricted_picking_type_code'))]"/>
+                                   attrs="{'invisible': [('hide_picking_type', '=', True)], 'readonly': ['&amp;', ('state', '!=', 'draft'), ('id', '!=', False)]}"/>
```

Comment:
Not sure about the feature. We will see with THD

Reply:
As discussed, I've reverted my changes in the search domain. However, I kept the condition allowing us to edit the operation type while in draft. I'll inform THD about this.

Reply:
THD confirmed that this is OK.